conda_build/metadata.py

```diff
@@ -2482,6 +2499,10 @@ def get_output_metadata(self, output):
             output_metadata.final = False
             output_metadata.noarch = output.get("noarch", False)
             output_metadata.noarch_python = output.get("noarch_python", False)
+            output_metadata.python_version_independent = (
+                output.get("python_version_independent")
+                or output_metadata.noarch == "python"
```

Comment:
Should this `or`-chain include `.noarch_python` too?

Reply:
Added